src/components/card/components/QuantitySection.jsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { useProductContext, useToast } from "../../../context";
+import { updateCartQuantity } from "../../../services";
+
+export const QuantitySection = ({ _id, qty }) => {
+  const {
+    productState: { cart },
+    productDispatch,
+  } = useProductContext();
+  const { showToast } = useToast();
+  return (
+    <div className="qty-sec flex-align-center">
+      <button
+        className={`btn btn-primary ${qty <= 1 && "disabled-btn"}`}
+        onClick={() => {
+          if (qty > 1)
+            updateCartQuantity({
+              _id,
+              action: { type: "decrement" },
+              productDispatch,
+              showToast,
+            });
+        }}
```

Comment:
also, handle the case if qty is 1, you can show some message to the user or you could remove the item from cart.

Reply:
okay thanks for the suggestion